pkg/zfs/zfs.go

```diff
@@ -114,7 +114,7 @@ func (z *zfsImpl) SetPermissions(dataset *Dataset) error {
 	if dataset.Mountpoint == "" {
 		return fmt.Errorf("undefined mountpoint for dataset: %s", dataset.Name)
 	}
-	cmd := exec.Command("update-permissions", dataset.Hostname, dataset.Mountpoint)
+	cmd := exec.Command("chmod", "g+w", dataset.Mountpoint)
```

Comment:
Is this a leftover of testing?
Because in the docker image this wouldn't work, as updating permissions is done over SSH (see https://github.com/ccremer/kubernetes-zfs-provisioner/blob/829d4a53de247938220ff9d3310bc5205bf88106/docker/update-permissions.sh)